swarm/pss/prox_test.go

```diff
@@ -361,6 +385,23 @@ func newProxServices(allowRaw bool, handlerContextFuncs map[Topic]handlerContext
 		return kademlias[id]
 	}
 	return map[string]simulation.ServiceFunc{
+		"bzz": func(ctx *adapters.ServiceContext, b *sync.Map) (node.Service, func(), error) {
+			// normally translation of enode id to swarm address is concealed by the network package
+			// however, we need to keep track of it in the test driver aswell.
+			// if the translation in the network package changes, that can cause thiese tests to unpredictably fail
+			// therefore we keep a local copy of the translation here
```

Comment:
Please don't do this. If we change the network package the test _ should_ fail if it doesn't conform to the new implementation.

Reply:
Ok never mind this. I will clean it up after the ENR sim changes merge. That will provide tools to do this properly.